code/datums/cache/crew.dm

```diff
@@ -18,50 +18,41 @@ GLOBAL_DATUM_INIT(crew_repository, /datum/repository/crew, new())
 	if(world.time < cache_entry.timestamp)
 		return cache_entry.data
 
-	var/tracked = scan()
-	for(var/obj/item/clothing/under/C in tracked)
+	for(var/mob/living/carbon/human/H in GLOB.mob_living_list)
```

Comment:
please update your PR and utilize the human list.

Reply:
Got it!